tests/lit-tests/ispc-intrinsics.ispc

```diff
@@ -0,0 +1,78 @@
+// RUN: %{ispc} %s --nowrap --target=host --emit-llvm-text --enable-llvm-intrinsics -o - 2>&1 | FileCheck %s
```

Comment:
Could you also add unknown intrinsic to the test?

Reply:
I have added a separate test `ispc-intrinsics-unknown.ispc`.